sci-libs/rdkit/rdkit-2015.03.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+
+inherit cmake-utils python-r1
+
+DESCRIPTION="Cheminformatics and machine-learning software written in C++ and Python"
+HOMEPAGE="http://www.rdkit.org/"
+SRC_URI="https://github.com/rdkit/rdkit/archive/Release_${PV//./_}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="+python -static-libs"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="dev-libs/boost
+	python? (
+		dev-libs/boost[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+	>=dev-db/sqlite-3"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}"/rdkit-Release_${PV//./_}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_PREFIX:PATH="${EPREFIX}"/usr
+		-DRDK_INSTALL_INTREE=0
+		-DRDK_BUILD_CPP_TESTS=OFF
+	)
+
+	if ! use static-libs; then
+		mycmakeargs+=( -DRDK_INSTALL_STATIC_LIBS=OFF )
+	fi
+
+	if ! use python; then
+		mycmakeargs+=( RDK_BUILD_PYTHON_WRAPPERS= )
```

Comment:
Oh no, copy paste error that I missed :disappointed: 

I'll send a PR for fixing it - but I don't entirely like the syntax, it will break for `+static-libs` if upstream ever sets `RDK_INSTALL_STATIC_LIBS=OFF` by default.

Is there any Gentoo function that just returns `ON` or `OFF` given a USE flag? I haven't seen it in the devmanual so far.